task_DEV_4/task_DEV_4/Rectangle.cs

```diff
@@ -1,40 +1,28 @@
 ﻿namespace task_DEV_4
 {
-    class Rectangle
+    class Rectangle : Square
     {
-        public int height;
-        public int width;
+        public double Height { get; set; }
+               
 
-        public Rectangle()
+        public Rectangle(double height, double side) : base(side)
         {
-
-        }
-
-        public Rectangle(int height, int width)
-        {
-            this.height = height;
-            this.width = width;
-        }
-
-        public void SetRectangleParameters(int height, int width)
-        {
-            this.height = height;
-            this.width = width;
+            Height = height;            
         }
 
-        public int CalculatePerimeter()
+        public void SetRectangleParameters(double height)
         {
-            return (height + width) * 2;
+            Height = height;            
         }
 
-        public int CalculateArea()
+        public override double CalculatePerimeter()
         {
-            return height * width;
+            return (Height * 2) + (FirstSide * 2);
```

Comment:
It can be rewritten like `2 * (Height + FirstSide)`

Reply:
Done